database/migrations/2020_03_05_021025_create_courses_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCoursesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('courses', function (Blueprint $table) {
+            $table->id();
+            $table->string('course_name');
+            $table->text('description');
+            $table->decimal('price', 3, 2);
```

Comment:
I think the 2nd argument of decimal is total length, inclusive of dollars. So you maximum price would be $9.99.

I would go for at least 7 or 8 instead.